www/ifarchive-commit

```diff
@@ -4,39 +4,95 @@ include_once "util.php";
 include_once "pagetpl.php";
 include_once "dbconnect.php";
 
-header("Content-type: text/plain");
+try {
+    $db = dbConnect();
 
-$db = dbConnect();
+    $ifdbid = get_req_data("ifdbid");
+    $newPath = get_req_data("path");
+    $apiKey = get_req_data("key");
 
-$id = mysql_real_escape_string(get_req_data("ifdbid"), $db);
-$newPath = mysql_real_escape_string(get_req_data("path"), $db);
-$oldUrl = "https://ifdb.org/ifarchive-pending?ifdbid=$id";
-$apiKey = get_req_data("key");
+    $oldUrl = "https://ifdb.org/ifarchive-pending?ifdbid=$ifdbid";
```

Comment:
does `$ifdbid` need to be url-quoted? I assume it's from a trusted source and guaranteed to be safe.

Reply:
It is from a trusted source, and we verify that by ensuring that the API key is correct.